strr-strata-web/app/pages/strata-hotel/dashboard/[registrationId].vue

```diff
@@ -1,60 +1,65 @@
-  } else {
-    // existing registration or application under the account
-    // set left side of header
-    title.value = strataBusiness.value.legalName
-    subtitles.value = [
-      strataBusiness.value.homeJurisdiction
-      // TODO: number of units
-      // t(`strr.label.listingSize.${platformDetails.value.listingSize}`)
-    ]
-    if (!isRegistration.value) {
-      setApplicationHeaderDetails(isRegistration.value, activeApplicationInfo.value?.hostStatus)
-    } else {
-      // @ts-expect-error - ts not picking up that it will have status attr in this case
-      setRegistrationHeaderDetails(activePlatform.value.status)
-    }
-    // TODO: strata side details
-    // setSideHeaderDetails()
-    // set sidebar accordian addresses
-    addresses.value = getDashboardAddresses(strataBusiness.value)
-    // set sidebar accordian reps
-    representatives.value = getDashboardRepresentives()
-    // update breadcrumbs with platform business name
-    setBreadcrumbs([
-      { label: t('label.bcregDash'), to: useRuntimeConfig().public.registryHomeURL + 'dashboard' },
-      { label: strataBusiness.value.legalName }
-    ])
-  }
-  loading.value = false
-})
+// onMounted(() => {
+//   loading.value = true
+//   const registrationId = route.params.registrationId as string
+//   loadStrata(registrationId)
+//   // set header stuff
+//   if (!activePlatform.value || !showStrataDetails.value) {
+//     // no registration or valid complete application under the account, set static header
+//     title.value = t('strr.title.dashboard')
+//     // TODO: verify for strata -- in this case it should always be a draft application
+//     todos.value = [getTodoApplication()]
+//   } else {
+//     // existing registration or application under the account
+//     // set left side of header
+//     title.value = strataBusiness.value.legalName
+//     subtitles.value = [
+//       strataBusiness.value.homeJurisdiction
+//       // TODO: number of units
+//       // t(`strr.label.listingSize.${platformDetails.value.listingSize}`)
+//     ]
+//     if (!isRegistration.value) {
+//       setApplicationHeaderDetails(isRegistration.value, activeApplicationInfo.value?.hostStatus)
+//     } else {
+//       // @ts-expect-error - ts not picking up that it will have status attr in this case
+//       setRegistrationHeaderDetails(activePlatform.value.status)
+//     }
+//     // TODO: strata side details
+//     // setSideHeaderDetails()
+//     // set sidebar accordian addresses
+//     addresses.value = getDashboardAddresses(strataBusiness.value)
+//     // set sidebar accordian reps
+//     representatives.value = getDashboardRepresentives()
+//     // update breadcrumbs with platform business name
+//     // setBreadcrumbs([
+//     //   { label: t('label.bcregDash'), to: useRuntimeConfig().public.registryHomeURL + 'dashboard' },
+//     //   { label: strataBusiness.value.legalName }
+//     // ])
+//   }
+
+//   loading.value = false
+// })
```

Comment:
was this failing? I hadn't tested it very thoroughly but I thought all this commented out stuff was working

Reply:
Yeah, in the base layer, the todoItems util uses the platform store, which doesn't exist in the strata app